README.md

```diff
@@ -56,77 +56,33 @@ make backend
    See the [Official installation instructions](https://istio.io/latest/docs/setup/getting-started/#install)
    for more information.
 
-2. Clone the apiclarity repository to your local system
+2. Add Helm repo
 
    ```shell
-   git clone https://github.com/apiclarity/apiclarity
-   cd apiclarity
+   helm repo add apiclarity https://apiclarity.github.io/apiclarity
    ```
 
-3. Deploy APIClarity in K8s. It will be deployed in a new namespace `apiclarity`:
+3. Deploy APIClarity with Helm
 
    ```shell
-   kubectl apply -f deployment/apiclarity.yaml
+   helm install --set 'global.namespaces={namespace1,namespace2}' apiclarity apiclarity/apiclarity -n apiclarity
    ```
+  **Note**:
+  namespace1 and namespace2 are the namespaces where the Envoy Wasm filters will be deployed to allow traffic tracing.
 
-   Note: The manifest uses `PersistentVolumeClaim`s to request two persistent
-   volumes. Make sure you have a default `StorageClass` available in your
-   cluster or, if deploying on a cluster that does not have this, edit the
-   manifest to provide your own local storage configuration.
```

Comment:
Do we want to delete this?

Reply:
storageClass is available in the values.yaml now.
This is usually obvious for a helm chart but we could keep it if you think it's necessary.